_posts/2022-12-08-try-out-webnn-api-early-using-webnn-polyfill.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: 'Try out WebNN API early using WebNN Polyfill'
+date: 2022-12-08 10:42:03 +0800
+author: Wanming Lin, Bruce Dai
+avatar: https://avatars.githubusercontent.com/u/379216?s=200&v=4
```

Comment:
@anssiko My question is can Wanming and Bruce represent W3C? Should it be replaced by Wanming's personal avatar?

Reply:
The reason why I used a W3C avatar is that I didn't find a way to add two avatars. :D

Reply:
Thanks @anssiko for the suggestion. :) 

It can be foreseen that there will be many tutorials, articles and blogs written for WebNN/WebML next year. Important matters will be published by you and Dominique on the W3C Blog and I think they will also be updated synchronously to https://webmachinelearning.github.io/blog. At present, I am not sure about the requirements for the number of multi-author articles, if the number of multi-authors increases, I will consider modifying the template to support it, just like https://pwadev.io/blog/the-china-pwa-developer-day-was-successfully-held (two authors) or even https://pwadev.io/blog/new-era-of-webassembly (three authors).

Reply:
I think multi-avatar is a nice to have feature. For the time being a satisfactory workaround is to use the `author` field for multiple authors.